polyply/__init__.py

```diff
@@ -23,10 +23,10 @@
     import atexit
     from contextlib import ExitStack
 except ImportError:
-    import os
-    DATA_PATH = os.path.join(os.path.dirname(__file__), 'data')
-    TEST_DATA = os.path.join(os.path.dirname(__file__), 'tests/test_data')
-    del os
+    from pathlib import Path
+    DATA_PATH = Path(__file__).parent / 'data'
+    TEST_DATA = Path(__file__).parent / 'tests/test_data'
```

Comment:
```suggestion
    TEST_DATA = Path(__file__).parent / 'tests' / 'test_data'
```

Reply:
Also in other places. Otherwise you lose the OS independence again.

Reply:
I changed all I found but might have missed some. That's fine though. Having the tests OS-independent is good to have but not 100% essential. We can catch the rest at a later stage.